packages/diagram/src/ui/top-left/TopLeftPanel.tsx

```diff
@@ -174,6 +176,13 @@ export const TopLeftPanel = () => {
         gap={'xs'}>
         {showNavigationButtons && <BackwardForwardButtons />}
         <ActionIconGroup className={css.actionIconGroup} orientation="vertical">
+          <Tooltip label="Search">
```

Comment:
I think we need to pass `enableSearch` from props to store, like:

https://github.com/likec4/likec4/blob/3c350b44f3602eaa3aaede2d57ff0e725b3d5c22/packages/diagram/src/LikeC4Diagram.tsx#L139

And show this icon if flag is set.

Reply:
I think I did what you wanted. I am not entirely certain whether I looked in the right places though.